index.bs

```diff
@@ -592,7 +592,6 @@ A [=sensor=] has an associated <dfn>current polling frequency</dfn> which is ini
 <pre class="idl">
 [SecureContext]
 interface Sensor : EventTarget {
-  readonly attribute SensorState state;
```

Comment:
Yeah, my worry about basing this on stats is that actual usage will depend highly on what documentation we release before we go to origin trial. @kenchris any thoughts?